includes/admin/upgrades/upgrade-functions.php

```diff
@@ -556,9 +600,140 @@ function edd_v21_upgrade_customers_db() {
 		// No more customers found, finish up
 
 		update_option( 'edd_version', preg_replace( '/[^0-9.].*/', '', EDD_VERSION ) );
+		delete_option( 'edd_doing_upgrade' );
 
 		wp_redirect( admin_url() ); exit;
 	}
 
 }
 add_action( 'edd_upgrade_customers_db', 'edd_v21_upgrade_customers_db' );
+
+/**
+ * Fixes the edd_log meta for 2.2.6
+ *
+ * @since 2.2.6
+ * @return void
+ */
+function edd_v226_upgrade_payments_db() {
+
+	global $wpdb;
+
+	if( ! current_user_can( 'manage_shop_settings' ) ) {
+		wp_die( __( 'You do not have permission to do shop upgrades', 'edd' ), __( 'Error', 'edd' ), array( 'response' => 403 ) );
+	}
+
+	ignore_user_abort( true );
+
+	if ( ! edd_is_func_disabled( 'set_time_limit' ) && ! ini_get( 'safe_mode' ) ) {
+		@set_time_limit(0);
+	}
+
+	$step   = isset( $_GET['step'] ) ? absint( $_GET['step'] ) : 1;
+	$number = 25;
+	$offset = $step == 1 ? 0 : ( $step - 1 ) * $number;
+
+	if ( 1 === $step ) {
+		// Check if we have any variable price products on the first step
+		$args = array(
+			'post_type'  => 'download',
+			'meta_key'   => '_variable_pricing',
+			'meta_value' => '1',
+			'posts_per_page' => -1
+		);
+
+		$sql = "SELECT ID FROM $wpdb->posts p LEFT JOIN $wpdb->postmeta m ON p.ID = m.post_id WHERE m.meta_key = '_variable_pricing' AND m.meta_value = 1";
```

Comment:
Yes the original thought was to use the download IDs found. But that logic was later ditched.